x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_selector/index.tsx

```diff
@@ -232,6 +232,7 @@ export const ConversationSelector: React.FC<Props> = React.memo(
                     e.stopPropagation();
                     onDelete(label);
                   }}
+                  data-test-subj="delete-option"
```

Comment:
Side note: It's this component that's the cause of alll the 

```
 console.error
    Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>.
    .....
 console.error
      Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    .....
```

errors in the tests. 

If we end up keeping this component after the upcoming design review, I think we'll need to do a full override when generating these options to get around the nested button behavior. I could've sworn this was keyboard nav friendly when I first put it together, but not seeing that work at the moment. It makes for a nice inline UX, but will need tweaked if we keep it around. This behavior is present in all the CRUD-friendly selectors, like the System and Quick Prompts.

Reply:
yeah i saw those all over the place, there are several within the assistant. figured we need a follow up for it https://github.com/elastic/security-team/issues/7804